web/interactive_terminal/README.md

```diff
@@ -0,0 +1,38 @@
+# The interactive `pandas` REPL
+
+An interactive REPL to easily try `pandas` in the browser, powered by JupyterLite.
+
+![image](https://user-images.githubusercontent.com/591645/175000291-e8c69f6f-5f2c-48d7-817c-cff05ab2cde9.png)
+
+## Build
+
+The interactive REPL is built with the `jupyter lite` CLI.
+
+First make sure `jupyterlite` and a kernel are installed:
+
+```bash
+python -m pip install jupyterlite
```

Comment:
Maybe the docs should mention `jupyterlite-core` to be consistent with the JupyterLite docs?

```suggestion
python -m pip install jupyterlite-core
```

Although for now in practice it does not change much, `jupyterlite` being a metapackage, it may include more dependencies in the future.